src/app/meditation/meditation.css

```diff
@@ -2,6 +2,31 @@ md-card.nopadding {
   padding: 0;
 }
 
+md-card-title {
+  display: flex;
+}
+
+.like-button {
+  width: 34px;
+  height: 34px;
+  font-size: 13px;
+  line-height: 13px;
```

Comment:
Could you add a small `margin-right` (like 10px worked for me) to align the button vertically with the like number of a meditation in the list?